spec/rubocop/cop/ezcater/rspec_require_gql_error_helpers_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+
+RSpec.describe RuboCop::Cop::Ezcater::RspecRequireGqlErrorHelpers, :config do
+  subject(:cop) { described_class.new }
+
+  let(:error_message) { described_class::MSG }
+  let(:expected_source) { "GraphQL::ExecutionError" }
+
+  context "when attempting to directly use GraphQL::ExecutionError" do
```

Comment:
do we want to include a spec for situations where no offense is registered?

Reply:
Did you have something specific in mind? I had tests for that previously, and it seemed like we all agreed that they weren't necessary. The tests for validating an offense is registered seem sufficient since they validate the expected source explicitly. Thoughts?

Reply:
Oh, I missed that conversation. I think it's fine, then!